codecov.yml

```diff
@@ -0,0 +1,11 @@
+coverage:
+  status:
+    project:
+      default:
+        target: auto
+        threshold: 5
+    patch:
+      default:
+        target: 75
+ignore:
+  - "test/"
```

Comment:
Why does this ignore all tests? Shouldn't it just be the e2e ones?

Reply:
This is not to ignore the e2e tests, it ignores coverage in the actual tests, ims has a similar thing for mock data, I initially had it to be for just the mock data as well which I can change it back to

Reply:
https://github.com/ral-facilities/inventory-management-system/blob/develop/codecov.yml

Reply:
Oh yeah, sorry you're right

Reply:
Feel free to merge.